Added hBP02 cards

jpantorilla opened the pull request:
- hBP02-014 to -017: "Shirogane Noel" Holomem cards
- hBP02-061 to -064: "Ninomae Ina'nis" Holomem cards
- hBP02-065 to -068: "Nerissa Ravencroft" Holomem cards
- hBP02-075: "Idol Felt Pen" Support (Item) card
- hBP02-076: "Custom PC" Support (Item) card
- hBP02-085: "HOLOLIVE FANTASY" Support (Event) card
- hBP02-086: "Holo Sparkling Wine" Support (Tool) card
- hBP02-100: "Shirogane Dan'in" Support (Fan) card

jpantorilla committed:
Added hBP02 cards

- hBP02-014 to -017, -085, -086, -100.

jpantorilla committed:
Added hBP02 cards.

- hBP02-061 to -064, -075.

jpantorilla committed:
Added hBP02 cards.

- hBP02-065 to -068.

jpantorilla committed:
Added hBP02-076.

daniel-k-taylor commented on globals/strings.gd:
```diff
@@ -239,6 +239,8 @@ func build_choose_cards_string(from_zone, to_zone, amount_min, amount_max,
 		"backstage":
 			to_zone_str = tr("YOUR_BACKSTAGE")
+		"bottom_of_deck":
+			to_zone_str = "%s of %s" % [tr("the bottom"), tr("YOUR_DECK")]
```
The %s of %s really needs to be translated for this to make sense in all languages.
We should probably just tr the whole phrase and not do a string format here.

daniel-k-taylor commented on globals/strings.gd:
```diff
@@ -422,6 +424,10 @@ func get_condition_text(conditions):
+			"attached_to_has_tags":
+				var inverse_str = "without" if condition.get("inverse") else "with"
+				var tags = condition["required_tags"]
+				text += "If attached to Holomem %s tags %s: " % [inverse_str, "/".join(get_tags_strings(tags))]
```
Can you please set this up so it can be tr'd better.
I'm guessing we'll want 2 separate strings for with/without.

Probably "If attached to Holomem with %s" or "If attached to Holomem without %s".

daniel-k-taylor commented on globals/strings.gd:
```diff
@@ -474,6 +480,12 @@ func get_condition_text(conditions):
+			"holomem_in_archive":
+				var amount_str = ""
+				if "amount_min" in condition and "amount_max" not in condition:
+					amount_str += "%s or more" % condition["amount_min"]
```
I'd really like to try to add tr to all new things we're adding.

daniel-k-taylor commented on globals/strings.gd:
```diff
@@ -518,7 +530,7 @@ func get_condition_text(conditions):
 				text += tr("Played a Support card this turn:") + " "
 			"self_has_cheer_color":
-				text += "" #"Has %s %s Cheer: " % [condition["amount_min"], "/".join(condition["condition_colors"])]
+				text += "Has %s %s Cheer: " % [condition["amount_min"], "/".join(get_color_strings(condition["condition_colors"]))]
```
can you tr and use the format for {Amount} {Color} like in other strings.

daniel-k-taylor reviewed (changes requested):
I'd prefer any new strings we add we try to make them translatable.

jpantorilla committed:
Changes based on suggestions.

- Moved to just use keywords instead of constructing word by word for translations.
